src/app/facility/utility-data/energy-consumption/utility-meter-data/edit-bill/edit-bill.component.html

```diff
@@ -33,4 +33,36 @@
     *ngIf="editMeter.source != 'Electricity' && (editMeter.scope == 5 || editMeter.scope == 6)"
     [editMeterData]="editMeterData" [addOrEdit]="addOrEdit" [editMeter]="editMeter" [meterDataForm]="meterDataForm"
     [invalidDate]="invalidDate">
-</app-edit-other-emissions-bill>
\ No newline at end of file
+</app-edit-other-emissions-bill>
+
+<div class="shadow-sm p-3 mb-3 mt-3 bg-light rounded" *ngIf="isElectron">
```

Comment:
Can we put the "Connect Utility Bill" content and it's corresponding .ts logic in it's own component please